sql/sql_parse.cc

```diff
@@ -1637,6 +1637,18 @@ bool dispatch_command(enum enum_server_command command, THD *thd,
   DEBUG_SYNC(thd,"dispatch_command_before_set_time");
 
   thd->set_time();
+
+  if (DBUG_EVALUATE_IF("mdev_15380_1", true, false))
+  {
+    thd->system_time.sec= 1523466002;
+    thd->system_time.sec_part= 799571;
+  }
+  if (DBUG_EVALUATE_IF("mdev_15380_2", true, false))
+  {
+    thd->system_time.sec= 1523466004;
+    thd->system_time.sec_part= 169435;
+  }
```

Comment:
Its better to use `sysvers_manual_timestamp` from c04c237a81b5b652e7b0096dac91c657d82361bb. If @vuvova gives permission to return it, please redo.